### ISSUE TITLE
Suggestion: Track Plugins in A Repository

### ISSUE THREAD

**Issue by chriscase** [wikkawik/WikkaWiki#1238]
I notice that all plugins are currently saved on the wikkawiki.org site.  I'd like to suggest creating a repository for plugins that you don't want to bundle with WikkaWiki.  That way we can get the benefits of Git, issue tracking, etc.  I would be happy to help with it if you're up for it.

Another alternative would be to bundle all of the known plugins with WikkaWiki and have a way of enabling/disabling them at will.

**Comment by klenwell** [comment]
:+1: for plugin repositories.

I'd recommend putting plugin repositories under @wikkawik account. Eventually, anyways. For official or vetted ones. 

Or perhaps just an empty `wikka-plugins` repository with a README that links to other plugin repositories. It could include recommendations for developers, too. Conventions for naming, coding, testing, README files, etc.

**Comment by chriscase** [comment]
Could have plugin developers issue pull requests just as with changes for the wiki, to add/modify plugins.

**Comment by bakoontz** [comment]
I agree with everything that's been said.  Pull requests are easy enough to deal with.  But there's no reason why we can't create our own action/handler repo as well.

Check your respective emails (hopefully I still have current emails for you both).
